tests/script-based-pre/playback_print/playback_print.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+# Test that concrete playback -Z concrete-playback does not override std print
+# functions
+
+set -o nounset
+
+function error() {
+    echo $@
+    # Cleanup
+    rm ${RS_FILE}
+    rm output.log
+    exit 1
+}
+
+RS_FILE="modified.rs"
+cp print_vars.rs ${RS_FILE}
+export RUSTFLAGS="--edition 2021"
+
+echo "[TEST] Generate test..."
+kani ${RS_FILE} -Z concrete-playback --concrete-playback=inplace
+
+echo "[TEST] Run test..."
+kani playback -Z concrete-playback ${RS_FILE} 2>&1 | tee output.log
+
+echo "------ Check output ----------"
+
+set -e
```

Comment:
Why not `set -e` earlier?

Reply:
Because Kani exit status is non-zero when the verification fail.